requirements-dev.in

```
@@ -12,6 +12,9 @@ setuptools==67.8.0;python_version>="3.12"
 # this is needed for test_patches
 dill~=0.3.3
 
+# this is needed for docs and test_version
+tomli;python_version<"3.11"
```

pro tip: it's nicer to have spacing in requirement specifiers. it's easier to read complex structures when they are sparse:
```suggestion
tomli; python_version < "3.11"
```